zappa/core.py

```diff
@@ -1258,6 +1278,12 @@ def rollback_lambda_function_version(self, function_name, versions_back=1, publi
         """
         response = self.lambda_client.list_versions_by_function(FunctionName=function_name)
 
+        # https://github.com/Miserlou/Zappa/pull/2192
+        if response['Versions'][-1]["PackageType"] == "Image":
+            raise NotImplementedError(
+                "Zappa's rollback functionality is not available for Docker based deployments"
+            )
+
```

Comment:
Rollbacks are not possible with docker based deployments. [`update_function_code`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html#Lambda.Client.update_function_code) requires an `ImageUri` for docker based deployments, but [`list_versions_by_function`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html#Lambda.Client.list_versions_by_function) does not return that...